charts/juicefs-csi-driver/templates/webhook.yaml

```diff
@@ -41,6 +41,42 @@ webhooks:
         juicefs.com/enable-injection: "true"
 {{- end }}
 
+{{- if eq .Values.mountMode "serverless" }}
```

Comment:
I guess the above `juicefs-admission-webhook` isn't accessed in this mode? if that's the case, `juicefs-admission-webhook` could be omitted?

Reply:
Yes, it can be omitted. They are doing the same thing, in a serverless case we only need `juicefs-admission-serverless-webhook`.